controllers/activemqartemis_controller_test.go

```diff
@@ -442,10 +442,26 @@ var _ = Describe("artemis controller", func() {
 			pdbObject := policyv1.PodDisruptionBudget{}
 			Eventually(func(g Gomega) {
 				g.Expect(k8sClient.Get(ctx, pdbKey, &pdbObject)).Should(Succeed())
+
+				resMap, err := getDeployedResources(createdCr, k8sClient)
```

Comment:
I think this use of getDeployedResources is too low level. Ideally these tests just work on the kube api.
A test that verifies that we can update the PDB when the CR updates may be at a better level of abstraction. Without the fix, an update will always fail b/c we see it as a create in error. 
A test that creates a CR with PDB, then updates the CR with a change that is relevant to the PDB and verifies that the PDB resource version is updated (!= to the original) will cover this nicely.

Reply:
yeah make sense. will add it.